packages/multichain-network-controller/src/MultichainNetworkController.ts

```diff
@@ -23,37 +23,56 @@ export type MultichainNetworkMetadata = {
   status: NetworkStatus;
 };
 
-export type MultichainNetworkConfiguration = {
+export type CommonNetworkConfiguration = {
+  /**
+   * EVM network flag.
+   */
+  isEvm: boolean;
+
+  /**
+   * The block explorers of the network.
+   */
+  blockExplorers: {
+    urls: string[];
+    defaultIndex: number;
+  };
+
   /**
    * The chain ID of the network.
    */
   chainId: CaipChainId;
+
   /**
    * The name of the network.
    */
   name: string;
+
   /**
    * The native asset type of the network.
    */
   nativeCurrency: CaipAssetType;
+};
+
+export type NonEvmNetworkConfiguration = CommonNetworkConfiguration & {
+  isEvm: false;
+};
+
+export type EvmNetworkConfiguration = CommonNetworkConfiguration & {
```

Comment:
I'm curious, do we want to add the EvmNetworkConfiguration type now l as a first step to migrate network controller to the multichain controller?

In this current version the only configuration we use is the non evm one

Reply:
It depends, in the ADR there is an option that proposes mirroring the EVM networks into this controller. Something to discuss further